test/Common/MoistThermodynamics/runtests.jl

```diff
@@ -181,6 +187,7 @@ end
       ts_dry = PhaseDry(e_int, ρ)
       ts_eq  = PhaseEquil(e_int, typeof(ρ)(0), ρ)
 
+      @test linearized_air_pressure(ts_eq, ρe_tot, Φ)  ≈ linearized_air_pressure(ts_dry, ρe_tot, Φ)
```

Comment:
You might also test that `linearized_air_pressure = air_pressure` for dry air with zero velocity.

Reply:
I'm not sure how to perform this test since `linearized_air_pressure` and `air_pressure` are not a function of velocity. Do you mean for zero gravitational potential energy? In that case, the comparison would be `air_pressure` against `ρ*R_d*(T_0 + e_tot/cv_d)`.

Reply:
I mean zero velocity, and said that because velocity enter `e_tot`. If the thermodynamic state does not depend on total energy (only internal energy), you simply do not need to worry about velocity. Then we should generally have `linearized_air_pressure = air_pressure` for dry air.

Reply:
I think I get it, and it looks a lot nicer now (the function signature is the same as `total_energy`), and the total energy is now consistent with the internal energy (I think that's what I was missing before).